kickstarts/base.ks.erb

```diff
@@ -157,6 +157,8 @@ open-vm-tools
 
 %post --log=/root/anaconda-post.log
 
+yum -y install http://www6.atomicorp.com/channels/atomic/centos/7/x86_64/RPMS/wmi-1.3.14-4.el7.art.x86_64.rpm
```

Comment:
It would be better to get this RPM from a non-commercial source (like EPEL), however I can't find it anywhere else.  @carbonin Is this the only place that you found it?  We may want to rebuild this in COPR if we can't find it anywhere else.